html/pfappserver/root/admin/nodes.tt

```diff
@@ -156,32 +247,64 @@ form {
                 </div>
                 <div class="tab-pane well" id="advanced">
                   <form id="advancedSearch" name="advancedNodeSearch" method="POST" class="form-inline" action="[% c.uri_for(c.controller('Node').action_for('advanced_search')) %]">
-                  <div>
-                        [% INCLUDE per_page_select.tt %]
-                  </div>
-                  <div>
-                    [% match_options = BLOCK %]
-                      <select name="all_or_any" class="input-mini">
-                        <option value="all">[% l('all') %]</option>
-                        <option value="any">[% l('any') %]</option>
-                      </select>
-                    [% END %]
-                    <br/>
-                    [% l('Match [_1] of the following conditions:', [match_options]) | none %]
-                  </div>
                   <br/>
-                  <div class="input-daterange datepicker">
-                    <label class="control-label">[% l('Detected Between :') %]</label>
-                    <input type="text" class="input-small" name="start" data-date-format="yyyy-mm-dd" />
-                    <i class="icon-arrow-right"></i>
-                    <input type="text" class="input-small" name="end" data-date-format="yyyy-mm-dd" />
-                    <a href="#last0day">[% l('Today') %]</a> | <a href="#last7days">[% l('7 days') %]</a> | <a href="#last30days">[% l('30 days') %]</a> | <a href="#last60days">[% l('60 days') %]</a>
-                  </div>
                   <table id="advancedSearchConditions" class="table-dynamic">
-                    <tbody>
+                    <thead data-no-remove="yes">
+                        <tr>
+                        <th colspan="2" class="pull-left">
+                          <div class="control-group">
+                            <div class="controls">
+                            [% INCLUDE per_page_select.tt %]
+                            </div>
+                          </div>
+                        </th>
+                        </tr>
+                        <tr>
+                          <th colspan="2" class="pull-left">
+                          <div class="control-group">
+                            <div class="input-daterange datepicker controls">
+                                <label class="control-label input-medium text-left">[% l('Online Between :') %]</label>
```

Comment:
Why a space before the semi-colon? "In print, a thin space is traditionally placed before a colon and a thick space after it. In modern English-language printing, no space is placed before a colon and a single space is placed after it. In French-language typing and printing, the traditional rules are preserved." - while I personally don't care too much, we should just be consistent across the whole app.